schema/games.yaml

```diff
@@ -40,12 +40,14 @@ schema;game:
         Sporadic: commits within the past year<br>
         Halted: no commits within the past year
     status:
+      required: True
       type: str
-      enum: ['playable', 'semi-playable', 'unplayable']
+      enum: ['playable', 'semi-playable', 'unplayable', 'N/A']
```

Comment:
Can the ommision of `status` serve as `'N/A'`?

Reply:
the build script checks this, but the add game forms can't do such dynamic validation. This change is so that the add game form also enforces status